Update color scheme and sidebar styles

jotvibfeng opened the pull request:
Refactored Bootstrap color variables and custom styles to use a new palette, affecting buttons, tables, alerts, and navigation elements. Improved sidebar and navigation appearance, added Roboto font, and enhanced header/sidebar SCSS for better UI consistency.

jotvibfeng committed:
Update color scheme and sidebar styles

Refactored Bootstrap color variables and custom styles to use a new palette, affecting buttons, tables, alerts, and navigation elements. Improved sidebar and navigation appearance, added Roboto font, and enhanced header/sidebar SCSS for better UI consistency.

Nikita-stud commented:
Good day, I see you have done two tickets and put them into one. This creates an issue. Please take a look at bootstrap in your code.

Nikita-stud commented on src/scss/styles.css:
We already have a dist/css with styles.css

jotvibfeng committed:
Add Bootstrap and custom stylesheets

Added Bootstrap 5.3.8 minified CSS and a custom style.css to the project. Updated index.html to include new stylesheets and modified related CSS and source map files to reflect these changes.

jotvibfeng committed:
Update style.css

Nikita-stud commented:
Overriden by New typography #129 merge
